### DISCUSSION TITLE
How to get the highest and lowest price TOMORROW?

### DISCUSSION THREAD

**Discussion by Matze-aaa** [mampfes/ha_epex_spot#161] [Q&A]
how can I get the highest and lowest price of tomorrow?
The sensors 
sensor.epex_spot_data_highest_price
sensor.epex_spot_data_lowest_price
are only for today.
Thx in advance!

**Comment by suaveolent** [comment]
See Readme - Service Calls.
For the tomorrow the offset needs to be set to 1.

**Reply by Matze-aaa** [reply]
Thx for your reply.
As I understand, the service call delivers the time of the highest /lowest price. 
I would like to know the value, how much is the kWh max and min tomorrow.

**Reply by suaveolent** [reply]
From the documentation:

The response contains the calculated start and end-time and the average price per kWh.
